pipeline/scraper/script.py

```diff
@@ -19,12 +19,21 @@ def main():
 
     today = datetime.datetime.now()
 
-    # Fetch applications from GitHub
+    # Fetch applications from GitHub for javascript
     for i in range(0, int(os.environ['MONTH'])):
         print("Fetching month " + str(i) + " of " + os.environ['MONTH'])
         run_query(month_calculation.month_delta(today, i))
 
     # Fetch package metadata from npmjs.com
     npm_query.run_query()
 
+    # Fetch applications from GitHub for python
+    for i in range(0, int(os.environ['MONTH'])):
+        print("Fetching month " + str(i) + " of " + os.environ['MONTH'])
+        run_query(month_calculation.month_delta(today, i), 'Python')
+
+    # Fetch package metadata from npmjs.com
+    npm_query.run_query()
```

Comment:
This won't work for Python packages. Probably need to rewrite `npm_query` to support multiple languages and fetch metadata from the right package repository (e.g. https://pypi.org and/or https://www.anaconda.com for Python)

Reply:
I will try pypi.org first.